### PR TITLE
Ad 2 connect data with api

### PR DESCRIPTION
**Pull request by agusss03**

### CODE REVIEW

File: App.tsx [hunk 1]
```diff
@@ -1,59 +1,49 @@
...
-    id: '3',
-    logo: require('./src/assets/images/xrp.png'),
-    name: 'XRP',
-    abr: 'XRP',
-    price: 0.220568,
-    status: -2.47,
-  },
-];
+const App = () => {
+  const [cryptos, setCryptos] = useState<any>([]);
```
Comment (tomasdematos) [inline App.tsx]: Don't use any type
create a type file to store the global types, in this case, the type for the crypto data:


```
src/
   types/
       crypto.ts
```

File: App.tsx [hunk 2]
```diff
@@ -1,59 +1,49 @@
...
-        <View>
-          <Title> CryptoTracker Pro </Title>
-        </View>
-        <ImageApp source={require('./src/assets/images/Face.jpg')} />
-      </Container>
-      <Space />
-      <View>
+  const getCryptos = async () => {
+    const response = await fetch(
+      'https://data.messari.io/api/v2/assets?fields=id,name,slug,symbol,metrics/market_data',
```
Comment (tomasdematos) [inline App.tsx]: "https://data.messari.io/api/v2/assets" Should be stored in a .env file, use this lib: 
https://yarnpkg.com/package/react-native-dotenv

File: src/components/Crypto/index.tsx [hunk 3]
```diff
@@ -19,36 +19,49 @@ type ItemProps = {
   crypto: {
```
Comment (tomasdematos) [inline src/components/Crypto/index.tsx]: the type for crypto, should be stored in: 
```
src/
   types/
      crypto.ts
```

File: App.tsx [hunk 4]
```diff
@@ -1,59 +1,49 @@
...
-            renderItem={({item}) => <Crypto crypto={item} />}
-          />
+          <View>
+            <FlatList
+              data={cryptos}
+              keyExtractor={({id}) => id}
+              renderItem={({item}) => <Crypto crypto={item} />}
+              ListFooterComponent={
+                <TouchableOpacity
+                  onPress={() => {
```
Comment (VicenBayugar) [inline App.tsx]: onPress={getCryptos}

**Pull request merged**